client/my-sites/customer-home/cards/actions/quick-links/index.jsx

```diff
@@ -54,66 +57,66 @@ export const QuickLinks = ( {
 	siteAdminUrl,
 	editHomePageUrl,
 	siteSlug,
+	blockEditorSettings,
+	areBlockEditorSettingsLoading,
 } ) => {
+	const isFSEActive = blockEditorSettings?.is_fse_active ?? false;
+
 	const translate = useTranslate();
 	const [
 		debouncedUpdateHomeQuickLinksToggleStatus,
 		,
 		flushDebouncedUpdateHomeQuickLinksToggleStatus,
 	] = useDebouncedCallback( updateHomeQuickLinksToggleStatus, 1000 );
 
+	const customizerLinks =
+		isStaticHomePage && canEditPages ? (
+			<ActionBox
+				href={ editHomePageUrl }
+				hideLinkIndicator
+				onClick={ trackEditHomepageAction }
+				label={ translate( 'Edit homepage' ) }
+				materialIcon="laptop"
+			/>
+		) : null;
+
 	const quickLinks = (
 		<div className="quick-links__boxes">
-			{ isStaticHomePage && canEditPages ? (
-				<>
-					<ActionBox
-						href={ editHomePageUrl }
-						hideLinkIndicator
-						onClick={ trackEditHomepageAction }
-						label={ translate( 'Edit homepage' ) }
-						materialIcon="laptop"
-					/>
-					<ActionBox
-						href={ `/page/${ siteSlug }` }
-						hideLinkIndicator
-						onClick={ trackAddPageAction }
-						label={ translate( 'Add a page' ) }
-						materialIcon="insert_drive_file"
-					/>
-					<ActionBox
-						href={ `/post/${ siteSlug }` }
-						hideLinkIndicator
-						onClick={ trackWritePostAction }
-						label={ translate( 'Write blog post' ) }
-						materialIcon="edit"
-					/>
-				</>
+			{ isFSEActive ? (
```

Comment:
I think we'd best add a capability check here, in line with the other links.

```suggestion
			{ isFSEActive && canManageSite ? (
```